pom.xml

```diff
@@ -39,7 +39,7 @@
     <powermock.version>2.0.9</powermock.version>
     <findbugs.annotations.version>3.0.1u2</findbugs.annotations.version>
 
-    <hazelcast.version>4.1</hazelcast.version>
+    <hazelcast.version>4.1.1-SNAPSHOT</hazelcast.version>
```

Comment:
Again, do we need the snapshot here?

Reply:
Using constant does not feel right actually 🙂 we do not need to concern about backward compatibility at master branch, right? Also we have released plugin with IMDG snapshot dependency before:
https://github.com/hazelcast/hazelcast-kubernetes/blob/v2.1/pom.xml#L42

Putting constant again and release `2.2.1` with `4.1` dependency is ok for me as well btw. After IMDG `4.1.1` release, we can upgrade IMDG dependency to `4.1.1` and start use `PartitionGroupMetaData.PARTITION_GROUP_NODE`.

Reply:
I think we still should care about the backward-compatibility because not all users use `hazelcast-all`. Now, for example, if they use `hazelcast:4.1` with `hazelcast-kubernetes:2.2`. And the see that `hazelcast-kubernetes:2.2.1` was released, they may want to update. But it'll crash their app. 

It's not actually about releasing from snapshot (because the dependency is `provided`), but about the code compatibility. 

We don't always have to keep backward-compatibility and we describe compatibilities [here](https://github.com/hazelcast/hazelcast-kubernetes#requirements-and-recommendations), but this change is actually so small that I'd favor keeping backward compatibility than reusing this one constant.

Reply:
> I think we still should care about the backward-compatibility because not all users use hazelcast-all. Now, for example, if they use hazelcast:4.1 with hazelcast-kubernetes:2.2. And the see that hazelcast-kubernetes:2.2.1 was released, they may want to update. But it'll crash their app.

I see, I totally missed out this scenario. I have reverted last commit 👍